packages/ingest/test/test-util.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const test = require('ava');
+const { conformProviderPath } = require('../util');
+
+test('conformProviderPath removes only leading slashes', (t) => {
+  const testString = '/////fake/path/';
+  const conformed = conformProviderPath(testString);
+  t.is(conformed, 'fake/path/');
+});
+
+test('conformProviderPath does not act on paths without leading slashes', (t) => {
+  const testString = 'fake/path/';
+  const conformed = conformProviderPath(testString);
+  t.is(conformed, 'fake/path/');
+});
+
+test('conformProviderPath returns empty string by default', (t) => {
+  let notString;
+  const conformed = conformProviderPath(notString);
+  t.is(conformed, '');
```

Comment:
Very nit-picky, but I think this would be just as clear _without_ the other variables.

```suggestion
  t.is(conformProviderPath(null), '');
```